cogs/remind_me.py

```diff
@@ -299,7 +311,8 @@ async def clear_reminders_backlog(self) -> None:
                     functools.partial(
                         lambda _user, _reminder: (
                             not _user.bot
-                            and DiscordMember.hash_discord_id(_user.id) == _reminder.discord_member.hashed_discord_id  # noqa: E501
+                            and DiscordMember.hash_discord_id(_user.id)
+                            == _reminder.discord_member.hashed_discord_id
```

Comment:
Comparison operators should not be split between multiple lines. The best you could do here is move the comparison (after the `and`) onto a new line, but that does not reduce the line length because the `and` + 1 space character is the same as the amount of indentation.